css-basics/css-basics_pseudo-park/css/styles.css

```diff
@@ -30,6 +31,14 @@
   */
 }
 
+.animal:nth-child(2) {
+  text-align: center;
+}
+.animal:nth-child(1),
```

Comment:
.animal:nth-child(1)  The crocodile should be on the left side, and it would be better to access elements by class selector